### DISCUSSION TITLE
Log Output Language

### DISCUSSION THREAD

**Discussion by wisegb** [dotnetcore/FreeSql#1078] [General]
Hi,

The log output seems to be in Chinese. Is there any way to set it to be in English?

Thanks
Bowden

**Comment by 2881099** [comment]
Yes, most of the errors are Chinese information, which can't be corrected for the time being.

If you encounter problems in use, please describe them clearly in issues. It's best to provide demonstration code.

What do you mean by log? We can internationalize it.
